api/envoy/config/listener/v3/udp_listener_config.proto

```diff
@@ -35,6 +36,8 @@ message UdpListenerConfig {
   // Configuration for QUIC protocol. If empty, QUIC will not be enabled on this listener. Set
   // to the default object to enable QUIC without modifying any additional options.
   QuicProtocolOptions quic_options = 7;
+
+  core.v3.TypedExtensionConfig udp_packet_packet_writer_config = 8;
```

Comment:
Worth documenting the absence case for QUIC and UDP proxy respectively?

Reply:
Done.

Reply:
Yes it would be great to get this working. I think I gave it a quick shot at some point but it was non-trivial because the code was inside quiche. Maybe it's easier now. I can't remember the details.

Reply:
The GSO code is still in QUICHE. I remember there was an excessive copy in the glue code, but it has been fixed. Since we have this config knob now, it will be easy for you to re-try again.

Reply:
After this PR, I'll make a stab at enabling GSO (via the QUICHE code, of course) for raw UDP.